deliver/lib/deliver/options.rb

```diff
@@ -136,6 +136,16 @@ def self.available_options
                                      verify_block: proc do |value|
                                        ENV["FASTLANE_ITC_TEAM_NAME"] = value
                                      end),
+        FastlaneCore::ConfigItem.new(key: :dev_portal_team_id,
+                                     short_option: "-s",
+                                     env_name: "DELIVER_DEV_PORTAL_TEAM_ID",
+                                     description: "The short ID of your team in the developer portal, if you're in multiple teams. Different from your iTC team ID!",
+                                     optional: true,
+                                     is_string: true,
+                                     default_value: CredentialsManager::AppfileConfig.try_fetch_value(:team_id),
+                                     verify_block: proc do |value|
+                                       ENV["FASTLANE_TEAM_ID"] = value.to_s
+                                     end),
```

Comment:
This option was already released publicly, and currently has a matching option still publicly available in `pilot`. We'll add it back in here again, even though it is unrelated to this fix.